### PR TITLE
feat(HACBS-1217): add unit testing for release functions

### PR DESCRIPTION
**Pull request by hongweiliu17**
Signed-off-by: Hongwei Liu hongliu@redhat.com

### CODE REVIEW

File: release/release_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+	k8sManager, _ := ctrl.NewManager(cfg, ctrl.Options{
+		Scheme:             clientsetscheme.Scheme,
+		MetricsBindAddress: "0", // this disables metrics
+		LeaderElection:     false,
+	})
+
+	k8sClient = k8sManager.GetClient()
+	go func() {
+		defer GinkgoRecover()
```
Comment (dirgim) [inline release/release_suite_test.go:95]: In order to resolve the unit testing issue, the releasePlan index cache needs to be set up - the following line needs to be added after this one:
```
Expect(snapshot.SetupReleasePlanCache(k8sManager)).To(Succeed())
```

Comment (hongweiliu17) [reply release/release_suite_test.go:95]: Thank you! It works.

**Review by dirgim — APPROVED** [review]
LGTM

**Review by srivickynesh — APPROVED** [review]
LGTM

**Pull request merged**